seqr/views/utils/file_utils.py

```diff
@@ -64,9 +68,9 @@ def parse_file(filename, stream):
 
 def _parse_excel_string_cell(cell):
     cell_value = cell.value
-    if cell_value and cell.data_type == 'n' and int(cell_value) == cell_value:
+    if cell_value is not None and cell.data_type == 'n' and int(cell_value) == cell_value:
```

Comment:
if `cell_value` is None than the part of this clause that is `cell.data_type == 'n'` will always fail, so adding the additional null check here is unnecessary. Just doing the single line change for `cell_value or ''` that I initially requested is really all you need to do

Reply:
bump

Reply:
The problem is when `cell_value` is `None`, `int(cell_value)` will cause an exception.

Reply:
If we do it as:
```
    if cell_value and cell.data_type == 'n' and int(cell_value) == cell_value:
```
Then `0` will be converted to `''`.